packages/taquito-tzip16/README.md

```diff
@@ -0,0 +1,19 @@
+# Taquito TZip-16 package
+
+`@taquito/tezbridge-tzip16` is an npm package that provides developers with TZip-16 functionality for Taquito. It can be injected as follows:
```

Comment:
provides the ability to retrieve metadata associated with a smart contract based on the TZIP-16 standard. The package also provides a way to execute the MichelsonStorageView found in the metadata

Reply:
added